scss/application.scss

```diff
@@ -24,11 +24,126 @@ body {
 }
 
 .page-content {
-  flex: 1;
   position: relative;
   overflow-y: auto;
+  flex-direction: column;
+  flex: 1;
+  padding: 50px;
+}
+
+.tech-card {
+  z-index: 3;
+  height: $tech-card-height + 10px;
+  width: $tech-card-width + 10px;
+  position: relative;
+  margin: 20px 20px;
+
+  &:hover {
+      height: $tech-card-height + 5px;
+      width: $tech-card-width + 5px;
+      margin-right: 25px;
+      margin-bottom: 25px;
+  }
+
+  &-background {
```

Comment:
There's no actual need for this since it's not obligatory for the child classes to start like it's parent class. You can name this like `.card-background` and it would be OK